stable/pipelines/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: pipelines
 home: https://jfrog.com/pipelines/
-version: 1.4.9
+version: 1.4.10
```

Comment:
Please change:
version: 1.4.10 -> 1.5.0
appVersion: 1.7.2 -> 1.8.0
so PR #1165 can be closed

Reply:
will do :)

Reply:
Done